elements/pfelement/src/pfelement.js

```diff
@@ -254,7 +467,7 @@ class PFElement extends HTMLElement {
         }
 
         if (hasPrefix) {
-          attrName = `${prefix}${attr}`;
+          attrName = `${tagPrefix}${attr}`;
```

Comment:
```suggestion
          attrName = `${prefix}-c-${attr}`;
```
?

Reply:
Good catch, that definitely shouldn't be `tagPrefix`, but it's not clear to me what it should be.  I renamed `prefix` to `tagPrefix` to differentiate the `pfe-` tag prefix from `pfe-c-` and `pfe-g-` attribute prefixes.  We could pull the attribute name from the properties system, but it seems wrong for `_mapSchemaToProperties`, whose job is to create properties based on a schema, to look at an alternative property system for attribute names. :thinking: 

Should we dodge this problem by removing `_mapSchemaToProperties`, since all elements will be migrated to the new properties system for 1.0?